member_go.py

```
@@ -0,0 +1,105 @@
+            else:
+                last_name= ""
+            name= (first_name + ' ' + last_name).strip()
+            writer.writerow([username,user.id,user.access_hash,name,target_group.title, target_group.id])      
+    print('Members scraped successfully from {} .'.format(filename))
+
+chat_list_index = list(range(len(chats)))
+
+for x in chat_list_index:
+    try: 
+        get(x)
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)